app-text/llpp/llpp-31.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs xdg-utils
+
+DESCRIPTION="Graphical PDF viewer which aims to superficially resemble less(1)"
+HOMEPAGE="https://github.com/moosotc/llpp"
+SRC_URI="https://github.com/moosotc/llpp/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+ocamlopt static-libs"
+
+RDEPEND=">=app-text/mupdf-1.15.0:0=[static-libs=]
+	media-libs/openjpeg:2[static-libs=]
+	media-libs/fontconfig:1.0[static-libs=]
+	media-libs/freetype:2[static-libs=]
+	media-libs/jbig2dec:=[static-libs=]
+	sys-libs/zlib[static-libs=]
+	virtual/jpeg:0[static-libs=]
+	x11-libs/libX11[static-libs=]
+	x11-misc/xsel"
+DEPEND="${RDEPEND}
+	app-text/asciidoc
+	virtual/pkgconfig
```

Comment:
asciidoc, pkgconfig are usually build-time dependencies which can be executed on CBUILD, so BDEPEND.

Is that the case here? Grepping the build system may be useful.

Reply:
That is the case here. The ebuild started as EAPI=4 where BDEPEND was not a feature. Moving.

dev-lang/ocaml should probably move there too – ocamlc gets executed in build time after all. I think I'll do that too.

Reply:
Just remember - DEPEND is build time too, but dependencies on CHOST (e.g. libraries we need to link against). This distinction matters for cross. BDEPEND is for executables where we don't care what arch they're running on, so we can run it on the host, e.g. for generating docs.

Reply:
Right. ocamlopt needs to run on the building host, but lablgl is linked against on the target architecture. I think it is mostly correct now.